gnd/src/main/java/com/google/android/gnd/ui/observationdetails/ObservationDetailsFragment.java

```diff
@@ -159,7 +163,18 @@ private void addField(Field field, Observation observation) {
                 binding.fieldValue.setVisibility(View.GONE);
                 binding.imagePreview.setVisibility(View.VISIBLE);
 
-                storageManager.loadPhotoFromDestinationPath(binding.imagePreview, value);
+                storageManager
```

Comment:
Same here - subscriptions should only be made inside lifecycle methods so they can be properly disposed of in their equivalent end lifecycle methods. To do this safely, you can create a PublishSubject in the view model that gets exposed to the fragment as a LiveData. Let's discuss if it's not clear how to do this here.

Reply:
Added a TODO